easybuild/cray_external_modules_metadata-20.05.cfg

```diff
@@ -0,0 +1,122 @@
+# metadata useful to EasyBuild (hpcugent.github.io/easybuild) for modules provided by Cray on Cray systems
+# authors: Guilherme Peretti-Pezzi (CSCS), Petar Forai (IMP/IMBA), Kenneth Hoste (HPC-UGent), Eirini Koutsaniti (CSCS)
+
+[cray-fftw]
+name = FFTW
+version = CRAY_FFTW_VERSION
+prefix = CRAY_FFTW_PREFIX
+
+[cray-hdf5]
+name = HDF5
+version = CRAY_HDF5_VERSION
+prefix = CRAY_HDF5_PREFIX
+
+[cray-hdf5-parallel]
+name = HDF5
+version = CRAY_HDF5_PARALLEL_VERSION
+prefix = CRAY_HDF5_PARALLEL_PREFIX
+
+[cray-libsci]
```

Comment:
Not sure if we need it. But if we do, why not also add `cray-libsci_acc`?

Reply:
Sure, I only added whatever was in the old metadata. As soon as dom is back I will check if the cray variables are in the module and if they are, I will add this as well

Reply:
I can add it to the metadata but what would be the equivalent `cray-libsci_acc` name for EasyBuild?

Reply:
There is none

Reply:
@gppezzi may know more

Reply:
I wouldn't add entries if we're not sure if they are actually needed, the only reference I see to `cray-libsci_acc` is on the Cosmo recipes. 

My guess is that it worked so far without this entry is because we don't need to explicitly rely on variables from `libsci_acc` (the cray wrappers are taking care of swapping the libraries), but I personally never used it so I would gladly take this statement back if you find an use case where just loading the libsci_acc module is not enough.